spark3-extensions/src/main/scala/org/apache/spark/sql/catalyst/parser/extensions/IcebergSqlExtensionsAstBuilder.scala

```diff
@@ -115,6 +116,18 @@ class IcebergSqlExtensionsAstBuilder(delegate: ParserInterface) extends IcebergS
       ctx.fieldList.fields.asScala.map(_.getText))
   }
 
+  /**
+   * Create a ENCRYPTED WITH (KEK | MEK) BY TABLE KEY command.
```

Comment:
Nit: You might want to reference the class that's created in the Scala doc, like the one below here.

Something like `Create a [[CreateTableEncKeyCommand]] for creating (applying?) a KEK or MEK encryption key for the table`.

The actual language is up to you as you, but it would be nice if the Scala Docs referenced the command class using `[[...]]`. 🙂

Reply:
Fixed. Thanks!
The majority of the visitXXX methods in this class don't use the `[[...]]`, though. Should I fix those too?